src/actions/images.ts

```
@@ -378,11 +378,30 @@ export const images_graph_data: GraphData = {
+              }
+            })();
+            if (referenceBeat === undefined) {
+              // error?
+              GraphAILogger.info(`reference beat not exist: index=${index}`);
             }
+            studioBeat.imageFile = referenceBeat.imageFile;
+            studioBeat.movieFile = referenceBeat.movieFile;
+            studioBeat.soundEffectFile = referenceBeat.soundEffectFile;
+            studioBeat.lipSyncFile = referenceBeat.lipSyncFile;
+            studioBeat.hasMovieAudio = referenceBeat.hasMovieAudio;
+            studioBeat.htmlImageFile = referenceBeat.htmlImageFile
```

ESLint

[error] Insert `;`

(prettier/prettier)